CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 0.21.0
+
+### Breaking Changes
+
+Version 0.21.0 reverts to version 0.17.0+1 with fixes to support Dart 2.
+Versions 0.18, 0.19, and 0.20 were effectively unsupported. This resolves the
```

Comment:
We'll be kicking ourselves in a week or a few if we don't get pedantic here: even though it will take a bit of work, I would want:

* What are the bullets from the CHANGELOG that are being reverted?
* What _new_ fixes made it into the internal branch that are being introduced here, one by one?

Reply:
https://github.com/dart-lang/observable/issues/51